_examples/docscansandbox/demo_test.go

```diff
@@ -0,0 +1,338 @@
+package docscansandbox
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/cucumber/godog"
+	"github.com/cucumber/messages-go/v10"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/check"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/check/report"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/filter"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/task"
+	_ "github.com/joho/godotenv/autoload"
+	"github.com/tebeka/selenium"
+	"github.com/tebeka/selenium/chrome"
+)
+
+const (
+	appBaseUrl        = "https://app:3000"
+	documentImagePath = "/usr/src/resources/image.jpg"
+)
+
+var pool = sync.Pool{
+	New: func() interface{} {
+		return startWebDriver()
+	},
+}
+
+func startWebDriver() selenium.WebDriver {
+	caps := selenium.Capabilities{"browserName": "chrome"}
+	caps.AddChrome(chrome.Capabilities{
+		Args: []string{
+			"--no-sandbox",
+			"--disable-dev-shm-usage",
+			"--disable-gpu",
+			"--window-size=1280,2000",
+			"--ignore-certificate-errors",
+		},
+	})
+	wd, err := selenium.NewRemote(caps, fmt.Sprintf("http://localhost:%d/wd/hub", 4444))
+	if err != nil {
+		panic(err)
+	}
+
+	// Set timeouts
+	var timeout time.Duration = 5000 * time.Millisecond
+	wd.SetPageLoadTimeout(timeout)
+	wd.SetImplicitWaitTimeout(timeout)
+	wd.SetAsyncScriptTimeout(timeout)
+
+	return wd
+}
+
+func newSandboxClient() *sandbox.Client {
+	key, err := ioutil.ReadFile(os.Getenv("YOTI_KEY_FILE_PATH"))
+	sdkID := os.Getenv("YOTI_SANDBOX_CLIENT_SDK_ID")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Might as well swap these lines around? If the code gets refactored, it makes it clear which error we're referring to
```suggestion
	if err != nil {
		panic(err)
	}
	sdkID := os.Getenv("YOTI_SANDBOX_CLIENT_SDK_ID")
```